library/src/main/java/org/xmtp/android/library/codecs/README.md

```diff
@@ -0,0 +1,71 @@
+# Adding specific content types to your application
+
+You can create any content type you want to be used in your application you can read me about creating content types here: [LINK]
+
+We have a few content types backed into the sdk directly for easy consumption (encryption and decryption) across applications.
+
+TextCodec - (always registered by default) allows the passing of plain text.
+AttachmentCodec - allows the passing of attachments
+RemoteAttachmentCodec - allows the passing of remote attachments
```

Comment:
I think we should make this a table that we update as we add more SDK supported content types.